README.md

```diff
@@ -55,7 +55,7 @@ technologies:
 - [Intel](https://www.intel.com)'s VT-x technology.
 - [ARM](https://www.arm.com)'s Hyp mode (virtualization extension).
 - [IBM](https://www.ibm.com)'s Power Systems.
-
+- [IBM](https://www.ibm.com)'s IBM Z mainframes.
```

Comment:
Should that be... ?

```suggestion
- [IBM](https://www.ibm.com)'s Z mainframes.
```

Reply:
We usually use IBM Z together. Otherwise I can add IBM Z without the the `'s`

Reply:
Ack. I vote for dropping the `'s`'s on one commit so you can they just add "IBM Z mainframes" on a second commit.

Reply:
I also prefer ;)